style.css

```diff
@@ -468,3 +502,242 @@ body {
     grid-template-columns: repeat(3, 1fr);
   }
 }
+
+@media screen and (min-width: 768px) {
+  header {
```

Comment:
- [ ] As the first reviewer required, please add a bottom shadow under your header and make it sticky to match the figma design:
![image](https://github.com/AbuTalha3/Portfolio-Setup/assets/91486791/4c56ab0e-939c-43b9-88a9-e386ea093ff3)